test/test-ffmpegCommandService.js

```diff
@@ -473,17 +473,17 @@ describe('FfmpegCommandService', function() {
 
             describe('Invalid video inputs', function() {
                 it('video array must contain at least one item', function() {
-                    validateCommonSchemaError({ video: [] }, 'child "video" fails because ["video" must be an object, "video" does not contain 1 required value(s)]');
+                    validateCommonSchemaError({ video: [] }, 'child "video" fails because ["video" does not contain 1 required value(s), "video" must contain at least 1 items]');
                 });
 
                 it('video.filePath must be provided', function() {
-                    validateCommonSchemaError({ video: { dimensions: { width: 1920, height: 1080 } } }, 'child "video" fails because [child "filePath" fails because ["filePath" is required], "video" must be an array]');
-                    validateCommonSchemaError({ video: [{ dimensions: { width: 1920, height: 1080 } }] }, 'child "video" fails because ["video" must be an object, "video" at position 0 fails because [child "filePath" fails because ["filePath" is required]]]');
+                    validateCommonSchemaError({ video: { dimensions: { width: 1920, height: 1080 } } }, 'child "video" fails because ["video" at position 0 fails because [child "filePath" fails because ["filePath" is required]], "video" at position 0 fails because ["0" must be an array]]');
```

Comment:
To be clear the original validation verified the json object, the new validation validates the internal structure it expects.

Reply:
Yea, this is probably ok at the end of the day since I'm not expecting a lot of external uses of this package.  However, the error messages should ideally match the input object, not the new mapped structure we create internally.

Reply:
I think I'm missing something with the comments here, what are referring to as the "json object" and what do we mean by "new mapped structure"?

Reply:
Original input for the video param for instance can be an object or an array of objects.  Before this PR, we used to first validate the input, then map onto a common interface, making sure only arrays were used afterwards.  With this PR, we first map onto the array structures, then validate.  Thus we don't validate the original input, but instead the modified input object.  Does that make sense?  I can walk over and chat in person if you want.